tools/dashboard/generate_dashboard.py

```diff
@@ -204,21 +292,23 @@ def gen_archive(config, log, outdir):
         # loop over all relevant commits
         all_url_rows = []
         all_html_rows = []
-        max_age = 1 + max([-1] + [log.index[sha] for sha in archive_reports.keys()])
-        for commit in log[:max_age]:
-            sha = commit["git-sha"]
+        max_age = 1 + max([-1] + [log.index_by_sha[sha] for sha in archive_reports])
```

Comment:
`max(-1, *(log.index_by_sha[sha] for sha in archive_reports))`

Reply:
Unfortunately, this does not work when the list is empty: 

```
    max_age = 1 + max(-1, *(log.index_by_sha[sha] for sha in archive_reports))
TypeError: 'int' object is not iterable

```